packages/prisma/migrations/20240320000001_add_booking_time_status_denormalized/migration.sql

```diff
@@ -0,0 +1,174 @@
+-- Create the denormalized table
+CREATE TABLE "BookingTimeStatusDenormalized" (
+    id INTEGER,
+    uid TEXT,
+    "eventTypeId" INTEGER,
+    title TEXT,
+    description TEXT,
+    "startTime" TIMESTAMP(3),
+    "endTime" TIMESTAMP(3),
+    "createdAt" TIMESTAMP(3),
+    location TEXT,
+    paid BOOLEAN,
+    status TEXT,
+    rescheduled BOOLEAN,
+    "userId" INTEGER,
+    "teamId" INTEGER,
+    "eventLength" INTEGER,
+    "timeStatus" TEXT,
+    "eventParentId" INTEGER,
+    "userEmail" TEXT,
+    "username" TEXT,
+    "ratingFeedback" TEXT,
+    "rating" INTEGER,
+    "noShowHost" BOOLEAN,
+    "isTeamBooking" BOOLEAN
+);
+
+-- Create indexes to match likely query patterns
+CREATE INDEX ON "BookingTimeStatusDenormalized" (id);
+CREATE INDEX ON "BookingTimeStatusDenormalized" ("userId");
+CREATE INDEX ON "BookingTimeStatusDenormalized" ("createdAt");
```

Comment:
```suggestion
CREATE INDEX "idx_booking_id" ON "BookingTimeStatusDenormalized" (id);
CREATE INDEX "idx_booking_user_id" ON "BookingTimeStatusDenormalized" ("userId");
CREATE INDEX "idx_booking_created_at" ON "BookingTimeStatusDenormalized" ("createdAt");
```

Lets make the indexes consistant with the named one below